cmd/agent/entrypoint.go

```diff
@@ -230,6 +233,9 @@ func (ep *Entrypoint) Start() error {
 	// signal is received.
 	signalHandler := signals.NewHandler(ep.cfg.Server.Log)
 
+	notifier := make(chan os.Signal)
+	signal.Notify(notifier, syscall.SIGHUP)
```

Comment:
Can you defer a call to `signal.Stop` and `close(notifier)` here?

Reply:
addressed